src/test/e2e/50_oci_package_test.go

```diff
@@ -196,7 +196,7 @@ func (suite *RegistryClientTestSuite) Test_5_Copy() {
 	}
 }
 
-func TestRegistryClientTestSuite(t *testing.T) {
+func TestRegistryClientSuite(t *testing.T) {
```

Comment:
I do think we should map out the OCI tests here and figure out which branches are being covered and which are not - extensions are definitely not covered and would be easy to break in current state.